projects/file-parsing/types-converting/json-struct.go

```diff
@@ -0,0 +1,33 @@
+package filesConverting
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+)
+
+type Score struct {
+	Name       string
+	High_Score int32
+}
+
+func JsonToStruct(textByte []byte) []Score {
+	result := []Score{}
+	s := Score{}
+
+	textTrimmed := strings.Trim(string(textByte), "[]\n")
+	textSliced := strings.Split(string(textTrimmed), "\n")
```

Comment:
This is a cute way of handling both JSON and repeated JSON in one way, but I can imagine some ways it could fail... Can you think what valid JSON could not work in your code? How could we handle that?